File: conteudo/dart/tmp.cpp
#include <iostream>

using namespace std;

void calcularImc(double peso, double altura)
{
    double imc = peso / (altura * altura);
    cout << "Peso = " << peso << endl;
    cout << "IMC = " << imc << "\n\n";
}

void calcularImc(double peso, double altura, string nome)
{
    double imc = peso / (altura * altura);

    cout << "Nome: " << nome << endl;
    cout << "Peso = " << peso << endl;
    cout << "IMC = " << imc << "\n\n";
}

int main()
{
    calcularImc(74, 1.82);
    calcularImc(75, 1.79, "Fulano");
    return 0;
}

File: conteudo/dart/tmp.dart
void main() {
  // Função anônima
  var imc = (peso, altura) {
    if (peso > 0 && altura > 0) return peso / (altura * altura);
    return 0.0;
  };

  print(imc(68, 1.73).toStringAsFixed(1));
}
